src/utils/time.ts

```diff
@@ -1,19 +0,0 @@
-import { MAX_BATCH_ID } from 'const'
-
-/**
- * Epoch in seconds
- */
-export function getEpoch(): number {
-  return Math.floor(Date.now() / 1000)
-}
```

Comment:
This is still useful/relevant, no?

Reply:
yep, it might, but I don't think we use it any more. I'll leave it just in case